homeassistant/components/demo/strings.select.json

```diff
@@ -0,0 +1,9 @@
+{
+  "state": {
+    "demo__speed": {
```

Comment:
I think that we discussed that we should change this to just be _. @bramkragten can you confirm.

Reply:
I've adjusted this PR to use `state._.{option}`, as there are no conflicting states in the demo example, that should be fine.

I've pushed a commit to the frontend part as well, to take `_` into account.

Reply:
Aaah right, seeing the build error now in hassfest (which didn't trigger on my pre-commit?)
I remember again.

<https://developers.home-assistant.io/docs/internationalization/core/#stringssensorjson>

So, _ is global for the entire Home Assistant codebase, as it results in a lookup for:

`component.select.state._.light_speed`

These translations are not bound by the integration domain, thus we required them to be namespaced.

For example:

`component.sensor.state.moon__phase.new_moon`

I'll revert my last commit in this PR. I'll leave the support on the frontend PR, as we could make some generic translations available for things like "on", "off", "auto" maybe... 🤷‍♂️

Reply:
OK confirmed with @bramkragten this is actually global, so the current method right now is correct.

We should definitely further discuss translation and how integration maybe can provide them better/easier (as shortly discussed on call just now). But that is a different cup of tea and probably out of scope for this PR.